src/pages/signIn/password/usePassword.tsx

```diff
@@ -48,18 +52,19 @@ export const usePassword = () => {
         await Promise.all(setStorages);
         const profileRes = await getUserProfileAdaptor();
         dispatch(setUserProfile(profileRes));
-        navigate(`/home`);
+        navigate(`/verifications`);
```

Comment:
if you wanna redirect to the default route please use "/" instead because later the default route will be changed (maybe)